build/board/EVK_iMX6ULL_512MB/Package/BootLoader/firmwareversions.log

```diff
@@ -0,0 +1,1478 @@
+U-Boot commit information:
```

Comment:
There are a lot of deltas here between upstream default branch and your version of U-Boot, Optee, and UEFI repos. Just checking to see is this intended?

Reply:
Forget to update this log file. Besides this change, another delta is U-Boot doesn't support iMX6ULL, have made a pull request https://github.com/ms-iot/u-boot/pull/10.

With the two pull requests, user can build UEFI and FFU for iMX6ULL in public preview branch.